update/upgrade_db.sh

```diff
@@ -9,12 +9,17 @@ HOST=localhost
 QWATSERVICE=qwat
```

Comment:
can you add a few lines on top to say what the file does (to distinct it from verify_upgrade)